ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/TypeList.java

```diff
@@ -20,14 +20,15 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableMap;
 import com.google.errorprone.annotations.Immutable;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import java.util.Map;
 import java.util.Objects;
 
 @Immutable
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class TypeList extends KsqlEntity {
 
-  private final ImmutableMap<String, SchemaInfo> types;
+  private final Map<String, SchemaInfo> types;
```

Comment:
Why was this changed to `Map`?

Reply:
Not 100%, but it broke the test. (Will revert.)